sync2jira/downstream_issue.py

```diff
@@ -960,12 +960,21 @@ def _update_github_project_fields(client, existing, issue, github_project_fields
     """
 
     for name, values in github_project_fields.items():
-        _, jirafieldname = values
-        try:
-            existing.update({jirafieldname: str(getattr(issue, name))})
-        except JIRAError:
-            # Add a comment to indicate there was an issue
-            client.add_comment(existing, f"Error updating GitHub project field")
+        fieldvalue = getattr(issue, name)
+        if name == 'storypoints':
+            jirafieldname =  next(iter(values['fieldmap'].values()))
+            try:
+                existing.update({jirafieldname: fieldvalue})
```

Comment:
Need to create a work around for empty values: https://jira.atlassian.com/browse/JSDSERVER-4364

Reply:
Update seems to be working: https://community.developer.atlassian.com/t/how-to-set-request-field-to-null-during-creation-when-a-default-value-is-defined/37425/2